qemu/tests/cfg/apicv_test.cfg

```diff
@@ -17,6 +17,7 @@
     Windows:
         netperf_client_link = "netperf.exe"
         client_path = "c:\\"
+        cmd_safe = false
```

Comment:
I prefer use the boolean `True/False`, not string `true/false` which need to convert by `get_boolean`.  It is up to you.

Reply:
My comment above is not correct, please ignore it~